packages/name-controller/src/NameController.ts

```diff
@@ -174,12 +171,10 @@ export class NameController extends BaseControllerV2<
   ): Promise<UpdateProposedNamesResult> {
     this.#validateUpdateProposedNamesRequest(request);
 
-    const chainId = this.#getChainId();
-
     const providerResponses = (
       await Promise.all(
         this.#providers.map((provider) =>
-          this.#getProviderResponse(request, chainId, provider),
+          this.#getProviderResponse(request, provider),
```

Comment:
`variation` missing on this jsdoc as well

Reply:
Done.